stack_overflow/src/main/java/com/gabrielcireap/stackOverflow/entity/Question.java

```diff
@@ -1,27 +1,33 @@
-package com.gabrielcireap.internet_banking_app.entity;
+package com.gabrielcireap.stackOverflow.entity;
 
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.ToString;
 
 import javax.persistence.*;
 import java.sql.Timestamp;
 import java.util.List;
 
 @Data
+@Entity
 @NoArgsConstructor
 @AllArgsConstructor
 public class Question {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Integer id;
+
+    @ManyToOne
+    @JoinColumn
     private User user;
     private String title;
     private String text;
     private Timestamp creationDate;
     private Integer voteCount;
+
+    @OneToMany(fetch = FetchType.EAGER)
+    @JoinTable(name = "questiontotags", joinColumns = @JoinColumn(name = "question_id"), inverseJoinColumns = @JoinColumn(name = "tag_id"))
```

Comment:
Small nitpicking regarding naming: would have been better to name the table `question_to_tag`.